.travis.yml

```diff
@@ -17,4 +18,5 @@ script:
   - DOCKER_CMD="docker run --rm --env-file /tmp/env_file -e GH_TOKEN -v $PWD:/git-restapi/ lmazuel/swagger-to-sdk"
   - if [[ $MODE == 'python' ]]; then $DOCKER_CMD AutorestCI/azure-sdk-for-python --pr-repo-id Azure/azure-sdk-for-python -o dev -v; fi
   - if [[ $MODE == 'node' ]]; then $DOCKER_CMD AutorestCI/azure-sdk-for-node --pr-repo-id Azure/azure-sdk-for-node -o master -v; fi
+  - if [[ $MODE == 'ruby' ]]; then $DOCKER_CMD AutorestCI/azure-sdk-ruby --pr-repo-id Azure/azure-sdk-ruby -o swagger-to-sdk -v; fi
```

Comment:
Yup I created that branch(swagger-to-sdk) to try this pipeline for ruby for now. Once i think, things are good we'd switch it to master branch.